host_source.go

```diff
@@ -835,11 +1090,28 @@ func refreshRing(r *ringDescriber) error {
 
 	r.session.metadata.setPartitioner(partitioner)
 	r.session.policy.SetPartitioner(partitioner)
+
+	return nil
+}
+
+// Experimental, this interface and use may change
+func refreshTablets(r *ringDescriber) error {
+	tablets, err := r.GetTablets()
+	if err != nil {
+		return err
+	}
+
+	r.session.ring.setTablets(tablets)
+	r.session.policy.SetTablets(tablets)
+
+	r.session.schemaDescriber.refreshTabletsSchema()
+
 	return nil
 }
 
 const (
 	ringRefreshDebounceTime = 1 * time.Second
+	tabletRefreshTime       = 1 * time.Second
```

Comment:
This should be changed to a larger value (1 minute?) - maybe even configurable value by a user.

Reply:
... but probably once CQL events start informing about tablet changes, this background refresher will get removed (so no need for user-configurable parameter).

Reply:
Changed, because of that tests are a little bit slow